sdk/appcenter-auth/src/test/java/com/microsoft/appcenter/auth/AuthTest.java

```diff
@@ -1398,6 +1384,24 @@ public void refreshTokenMultipleTimes() throws Exception {
         verify(mAuthTokenContext, never()).setAuthToken(isNull(String.class), isNull(String.class), isNull(Date.class));
     }
 
+    @Test
+    public void refreshTokenWithoutConfig() {
+        ArgumentCaptor<AuthTokenContext.Listener> authTokenContextListenerCaptor = ArgumentCaptor.forClass(AuthTokenContext.Listener.class);
+        doNothing().when(mAuthTokenContext).addListener(authTokenContextListenerCaptor.capture());
+
+        /* Start auth service. */
+        Auth auth = Auth.getInstance();
+        start(auth);
+
+        /* Mock foreground. */
+        auth.onActivityResumed(mock(Activity.class));
+
+        /* Connect to online with token to update. */
+        authTokenContextListenerCaptor.getValue().onTokenRequiresRefresh("accountId");
+
+        /* Do not crash. */
```

Comment:
Is this comment necessary here?

Reply:
That describes why there is no any verifies here

Reply:
I would change the method name to indicate that then - see suggestion